### DISCUSSION TITLE
Smart Tables

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#762] [Feature Requests]
Creating tables can be quite a pain. IAWriter recently implemented support for "Smart Tables" which makes editing so much easier. We could implement something similar.

https://ia.net/writer/blog/syntax-control-individual-scroll-snippets

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by Dieterbe** [comment]
Looks like what you're suggesting is this:
![IAWriter example](https://ia.net/wp-content/uploads/2020/07/iA-Writer-Windows-Tables.gif)
This looks like it would be useful in the **raw editor** of gitjournal, when you're working with the raw markdown directly.

But on mobile I would prefer to have a more visual experience (I guess this is what the **markdown editor** is for but i'm not sure because i expected to see helpers to make text bold, italic, etc and they don't seem to be there). I think Google docs solves this really nicely and to have a similar experience to this would be really nice.
![google-docs-table-1](https://user-images.githubusercontent.com/20774/140638023-eada331b-2aa1-4865-9026-cf368202aa6f.png)

![google-docs-table-2](https://user-images.githubusercontent.com/20774/140638022-228fb7cd-95e3-4f03-b3c1-534469f9fe24.png)
